tests/posit/connect/__api__/v1/users.json

```diff
@@ -0,0 +1,19 @@
+{
```

Comment:
I'd delete this and just use `tests/posit/connect/__api__/v1/users?page_number=1&page_size=2.json` in the test